modules/tensor_mechanics/test/tests/ad_action/tests

```diff
@@ -0,0 +1,57 @@
+[Tests]
+  issues = '#7555'
+  [./two_block_new]
+    type = Exodiff
```

Comment:
Is this action setting up objects that haven't been AD'd? E.g. can we or can't we add a `PetscJacobianTester`?

Reply:
Yeah, this should have jacobian tests...